src/control/lib/daos/pool_property.go

```diff
@@ -340,6 +340,61 @@ func PoolProperties() PoolPropertyMap {
 				},
 			},
 		},
+		"checkpoint": {
+			Property: PoolProperty{
+				Number:      PoolPropertyCheckpointMode,
+				Description: "WAL Checkpointing behavior",
+			},
+			values: map[string]uint64{
+				"disabled": PoolCheckpointDisabled,
+				"timed":    PoolCheckpointTimed,
+				"lazy":     PoolCheckpointLazy,
+			},
+		},
+		"checkpoint_freq": {
+			Property: PoolProperty{
+				Number:      PoolPropertyCheckpointFreq,
+				Description: "WAL Checkpointing frequency",
+				valueHandler: func(s string) (*PoolPropertyValue, error) {
+					rbErr := errors.Errorf("invalid Checkpointing Frequency value %s", s)
+					rsPct, err := strconv.ParseUint(strings.ReplaceAll(s, "%", ""), 10, 64)
+					if err != nil {
+						return nil, rbErr
+					}
+					return &PoolPropertyValue{rsPct}, nil
+				},
+				valueStringer: func(v *PoolPropertyValue) string {
+					n, err := v.GetNumber()
+					if err != nil {
+						return "not set"
+					}
+					return fmt.Sprintf("%d", n)
+				},
+				valueMarshaler: numericMarshaler,
+			},
+		},
+		"checkpoint_thresh": {
+			Property: PoolProperty{
+				Number:      PoolPropertyCheckpointThresh,
+				Description: "WAL Checkpointing trigger threshold",
+				valueHandler: func(s string) (*PoolPropertyValue, error) {
+					rbErr := errors.Errorf("invalid Checkpointing threshold value %s", s)
+					rsPct, err := strconv.ParseUint(strings.ReplaceAll(s, "%", ""), 10, 64)
+					if err != nil {
+						return nil, rbErr
+					}
+					return &PoolPropertyValue{rsPct}, nil
+				},
```

Comment:
I guess this is actually a percentage? In any case, might want to make the description more descriptive.